### ISSUE TITLE
`<` and `>` character are prefixed with `\` in method definitions

### ISSUE THREAD

**Issue by nicfv** [typedoc2md/typedoc-plugin-markdown#564]
I brought up an issue in #557 but it was overlooked and then the issue was closed.

When using generic types in class methods, this will add `\` characters before the `<` and `>`. See example:

```ts
export class Example<T> {
    constructor() { }
    /**
     * Get the first item in `arr`
     */
    public first(arr: T[]): T {
        return arr[0];
    }
}
```

Which yields this markdown file:

```md
[min-example](../README.md) / [Exports](../modules.md) / Example

# Class: Example\<T\>

## Type parameters

| Name |
| :------ |
| `T` |

## Table of contents

### Constructors

- [constructor](Example.md#constructor)

### Methods

- [first](Example.md#first)

## Constructors

### constructor

• **new Example**\<`T`\>(): [`Example`](Example.md)\<`T`\>

#### Type parameters

| Name |
| :------ |
| `T` |

#### Returns

[`Example`](Example.md)\<`T`\>

#### Defined in

index.ts:2

## Methods

### first

▸ **first**(`arr`): `T`

Get the first item in `arr`

#### Parameters

| Name | Type |
| :------ | :------ |
| `arr` | `T`[] |

#### Returns

`T`

#### Defined in

index.ts:6
```

Which mostly makes sense, but when combining with `mkdocs`, I get these weird artifacts:

![image](https://github.com/tgreyuk/typedoc-plugin-markdown/assets/24358775/a9cc5611-bcd2-4bfb-97c5-44d90b4d5601)

Is it possible to create an option on how `<` and `>` are escaped? Or better yet, turn `<` into `&lt;` and `>` into `&gt;`. I tried replacing those characters with the HTML entities, and it worked perfectly.

**Comment by nicfv** [comment]
I just installed `4.0.0-next.47` by the way. It is amazing, great job, and thank you!

Unfortunately, I'm still experiencing this bug. :(

**If** not fixed, and someone comes across this post in the future, I added these lines to my workflow which _seem_ to fix the problem. (These lines search and replace all instances of `\<` with `&lt;` and `\>` with `&gt;` in all markdown files in the `docs/` directory.)

```shell
sed -i 's|\\<|\&lt;|g' docs/**/*.md
sed -i 's|\\>|\&gt;|g' docs/**/*.md
```

**Comment by tgreyuk** [comment]
thanks - i'll have a think about the best solution is for this.

**Comment by nicfv** [comment]
Thank you for taking a look. I also created issue https://github.com/mkdocs/mkdocs/issues/3563 to see if they have anything to add to it.

**Comment by tgreyuk** [comment]
"useHTMLEncodedBrackets" option will html encode angle brackets (rather than escape backslash). Available in typedoc-plugin-markdown@4.1.2.
